[JsonPath transform] Improve/clarify README (#6904)
openhab/openhab-addons: bundles/org.openhab.transform.jsonpath/README.md

 end
```

Now the resulting Number can also be used in the label to [change the color](https://docs.openhab.org/configuration/sitemaps.html#label-and-value-colors) or in a rule as a value to compare.

## Differences to standard JsonPath

Compared to standard JsonPath, the transformation returns single values instead of arrays.
The transformation also will not ever return `null`.
This makes it possible to use the transform in labels or output channels of Things.

> Because the transformation will not return arrays or null values, if the JsonPath expression provided results in no matches, the transformation will return the entire original JSON string.

Reviewer: ```suggestion
If the JsonPath expression does not match (or the match is invalid), the entire original JSON string is returned.
```
